src/test/java/io/openliberty/tools/intellij/it/TestUtils.java

```diff
@@ -388,13 +388,12 @@ public static void validateTestReportExists(Path pathToTestReport) {
                     Thread.sleep(retryIntervalSecs * 1000);
                 } catch (Exception e) {
                     e.printStackTrace(System.out);
-                    continue;
                 }
-                continue;
+            } else {
+                return;
             }
-
-            return;
         }
+        throw new IllegalStateException("Timed out waiting for test report: " + pathToTestReport + " file to be created.");
```

Comment:
Thanks for the demo this morning. This looks correct, but will wait for the go-ahead from @TrevCraw that `main` is open again for PR merges before approving.

Reply:
Sure @mrglavas , we can wait. Thank You

Reply:
`main` is open again, so I'm planning to approve now.

Reply:
Thanks @mrglavas